easybuild/easyconfigs/c/cuDNN/cuDNN-7.5.0.56-CUDA-10.0.130.eb

```diff
@@ -0,0 +1,34 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Author:    Stephane Thiell <sthiell@stanford.edu>
+##
+easyblock = 'Tarball'
+
+name = 'cuDNN'
+version = "7.5.0.56"
+cuda_version = '10.0.130'
+
+cuda_version_major_minor = '.'.join(cuda_version.split('.')[:2])
```

Comment:
@henkela If you still want to get this merged, you'll need to rename the local variables (`cuda_version` and `cuda_version_major_minor`) to `local_...` to make the (updated) tests in Travis pass; see also https://easybuild.readthedocs.io/en/develop/Easyconfig-files-local-variables.html